app/assets/javascripts/discourse/lib/lazy-load-images.js

```diff
@@ -84,6 +84,14 @@ function show(image) {
   }
 }
 
+function forEachImage($post, callback) {
+  $("img", $post).each((_, img) => {
```

Comment:
can't this be:

```
$post[0].querySelectorAll("img").forEach(xxx
```

Reply:
Nice! I removed a lot of JQuery but missed this one. Updated :+1: